amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/impl/InMemoryExchange.java

```diff
@@ -44,11 +44,20 @@ public class InMemoryExchange extends AbstractAmqpExchange {
     private final long currentLedgerId;
     private long currentEntryId;
 
+    private Map<String, Object> properties;
```

Comment:
Maybe we can add this field in class `AbstractAmqpExchange`.

Reply:
`PersistentExchange` don't need this field.